core/storage/user/gae_models.py

```diff
@@ -97,6 +97,85 @@ def get_by_normalized_username(cls, normalized_username):
             cls.normalized_username == normalized_username).get()
 
 
+class ActivitiesCompletedByLearnerModel(base_models.BaseModel):
```

Comment:
There are two inconsistencies between the name of this class and the name of the class that follows. Firstly, this class has a "ByLearner" part; secondly, the adjective and noun are swapped. It might be a good idea to make them more consistent.